.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
```

Comment:
Only possibility changes are update in previous lambdas or create a new one so remove the default checklist and add this
```md
- [ ] Update lambda @<name of lambda>
- [ ] Created new lambda @<name of lambda>
- [ ] Other 
```